### ISSUE TITLE
Problem in example for a custom suppressKeyboardEvent

### ISSUE THREAD

**Issue by Eugeno** [ag-grid/ag-grid#8392]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```

**Providing a Reproducible Scenario**
https://www.ag-grid.com/angular-data-grid/keyboard-navigation/#custom-cell-component
If we leave only one focusable element in the cell, shift+tab doesn't work.

But it can be fixed by moving string `const isFirstChildFocused = firstCellChildEl && document.activeElement === firstCellChildEl;` above the string `if (!cellHasFocusedChildren) {`.

**Comment by AG-Zoheil** [comment]
Hello 

Thank you for reporting this - much appreciated!

Based on your description we've now reproduced this issue as a bug.

We have added this bug to our backlog and we are tracking it with the following reference:  **AG-12434**

We try to fix bugs from one release to the next, so this should be fixed in the next release or the one after if it was raised too close to the next release date.

You can follow the status of this item on the product pipeline page:
https://ag-grid.com/pipeline/

The best way to track this is to sign up for AG Grid new release notifications using [the instructions here](https://ag-grid.zendesk.com/hc/en-us/articles/360002648058-How-do-I-sign-up-for-ag-Grid-release-notifications-). This way you'll know as soon as a new version is out and you can check whether this specific item was implemented on [the changelog page](https://www.ag-grid.com/ag-grid-changelog/).

Thanks again for bringing this up with us.

Kind regards,
Zoheil

**Comment by AG-Zoheil** [comment]
Hi @Eugeno,

Thank you for raising this on GitHub.

We've reviewed this and do not believe this is a bug. The example code has been written to show how to handle multiple elements and works for the example provided in the docs and not for other use cases.

Kind regards,
Zoheil

**Comment by Eugeno** [comment]
Yes, this code works for the example with multiple elements, but my proposed change makes the code more universal. This could be helpful for someone who encounters a situation similar to mine, saving them from having to debug it.
